public/components/cards/27.html

```diff
@@ -0,0 +1,48 @@
+<a class="block bg-[#fefefe] shadow-xl rounded-xl text-center p-5" href="">
```

Comment:
Can you use a Tailwind CSS class rather than an arbitrary value for the background color, please?

Also, let's try `p-6`, I think that will look a little nicer.

Reply:
Let's try `bg-gray-100` here.